### ISSUE TITLE
how to add a callback function for saving loss during the training

### ISSUE THREAD

**Issue by hangwudy** [matterport/Mask_RCNN#1296]
Hello everyone, 

When training the model, I can see the losses going down.
I want to log these losses to a file over time so that I can draw a loss/accuracy graph.
I looked for some solutions on the internet, but they are all for model.fit() function.
Here we have our custom function mode.train().
So, I would like to ask, how can I add a call back function to save the loss over time?
Thanks!!

**Comment by Soufiane1503** [comment]
Hi,
It's very easy for the loss graph, Just use Tensorboard. 
Activate your environment on a terminal and then in your project directory run:
                    tensorboard --logdir [your MODELS directory]
It provides an URL like: http://localhost:6006/  where you can find your loss graph.

**Comment by hangwudy** [comment]
> Hi,
> It's very easy for the loss graph, Just use Tensorboard.
> Activate your environment on a terminal and then in your project directory run:
> tensorboard --logdir [your MODELS directory]
> It provides an URL like: http://localhost:6006/ where you can find your loss graph.

Thank you very much for your advice, I will try it right away.

**Comment by khanfarhan10** [comment]
Could you tell me how to do this on Google Colab. I am training on my custom dataset there.
